masonry/src/passes/event.rs

```diff
@@ -181,6 +207,15 @@ pub(crate) fn run_on_text_event_pass(root: &mut RenderRoot, event: &TextEvent) -
             }
             handled = Handled::Yes;
         }
+
+        if key.physical_key == PhysicalKey::Code(KeyCode::F11)
+            && key.state == ElementState::Pressed
+            && handled == Handled::No
+        {
+            root.global_state.inspector_state.is_picking_widget =
+                !root.global_state.inspector_state.is_picking_widget;
+            handled = Handled::Yes;
+        }
     }
```

Comment:
Would it be worth having a similar keybinding for toggling debug paint? That's probably a follow-up/different PR.